src/python/pants/backend/project_info/peek.py

```diff
@@ -124,17 +121,17 @@ async def get_target_data(
             Targets,
             DependenciesRequest(tgt.get(Dependencies), include_special_cased_deps=True),
         )
-        for tgt in targets_with_dependencies
+        for tgt in sorted_targets
     )
     hydrated_sources_per_target = await MultiGet(
         Get(HydratedSources, HydrateSourcesRequest(tgt[SourcesField]))
         for tgt in targets_with_sources
     )
 
-    expanded_dependencies_map = {
-        tgt.address: tuple(dep.address.spec for dep in deps)
-        for tgt, deps in zip(targets_with_dependencies, dependencies_per_target)
-    }
+    expanded_dependencies = [
+        tuple(dep.address.spec for dep in deps)
+        for tgt, deps in zip(sorted_targets, dependencies_per_target)
+    ]
```

Comment:
@stuhood `tgt` is unused in this comprehension, so no need to `zip` in `sorted_targets` (or is there another mistake hiding in here)?